modules/create_timepoint/jsx/createTimepointIndex.js

```diff
@@ -184,10 +190,13 @@ class CreateTimepoint extends React.Component {
    * Cohort refreshes when Project changes.
    */
   handleCohort() {
+    const {t} = this.props;
     const state = Object.assign({}, this.state);
     if (Array.isArray(state.storage.cohort[state.form.value.project])) {
       // Display error message to user.
-      const errorMessage = `No cohorts defined for project: ${
+      const errorMessage = t(
+        'No cohorts defined for project:',
+        {ns: 'create_timepoint'}) + ` ${
```

Comment:
These should be handled with variables in the template string, not string concatenation.

ie something like (untested).
```
t('No cohorts defined for project: {{project}}', 
{ns: 'create_timepoint',
project: this.state.form.options.project[
          this.state.form.value.project
        ]});
```